src/api.js

```diff
@@ -54,6 +56,21 @@ async function prepare( cmd ) {
 	};
 }
 
+function json( set ) {
+	if ( set.length === 0 ) {
+		return null;
```

Comment:
When I was reading through the tests, this seemed a bit surprising as I was expecting it to behave more like `query` not `queryFirst` and return an empty array here.  My thinking is that all of the `query*` functions indicate how that function is going to handle the results differently than just plain old `query`.  Honestly it isn't a big deal either way, just thought I would relay my thoughts as I was thinking about consuming this API.

Reply:
Dropping JSON support for now so we can rethink this.